### PR TITLE
Revised staking operations doc

### PR DESCRIPTION
**Pull request by rajkaramchedu**
A bunch of updates to the staking operations doc, including Slack thread discussion extracts, insights and gotchas. Updated the related docs also. 

@sherry-x this below especially needs your attention:
- I generalized the Connecting to the network section with "mainnet" as default. Hence I removed steps that ask the user to pull the main branch at specific commit hash, and specific Docker image ID. Wouldn't these be default default on the main branch for mainnet? I did retain the instruction to change the chain ID and set it to 1. 
- Other changes are not invasive like the above. If these changes are okay, then we should land this asap.
- **Question**: I need to move the **Add monitoring components** from Shutting Down section. I think this belongs in the validator installation section. Can you suggest a place? 
`+` @movekevin 
- I have not explained anything on "pool_type". Should we add the explanation for StakingContract, Vesting and Direct options?
- Other than the above, after I make fixes based on review comments, can we make the staking pool operations doc final? I will submit a separate PR for staking concept doc updates.

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4896)

### CODE REVIEW

File: developer-docs-site/docs/nodes/validator-node/operator/shutting-down-nodes.md [hunk 1]
```diff
@@ -5,15 +5,22 @@ slug: "shutting-down-nodes"
...
 # Shutting Down Nodes
 
-Follow these instructions to shut down the validator node and cleanup the resources used by the node.
+Follow these instructions to shut down the validator node and validator fullnode, and cleanup the resources used by the nodes.
 
-:::tip Leave validator set first
-Before you shutdown the node, make sure to first leave validator set first. This will be become effective in the next epoch.
+## Leaving validator set
+
+Before you shutdown the node, make sure to first leave validator set first. This will be become effective in the next epoch. Also note that a node can choose to leave validator set at anytime, or it would happen automatically when there is insufficient stake in the validator account. To leave the validator set, run the below command, shown using the example profile of `testnet-operator`:
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/operator/shutting-down-nodes.md:12]: first the validator set.

File: developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md [hunk 2]
```diff
@@ -5,23 +5,118 @@ slug: "staking-pool-operations"
...
+The below CLI command examples use mainnet. See the `--rest-url` value for testnet or devnet in [Aptos Blockchain Deployments](/docs/nodes/aptos-deployments.md).
 :::
 
-The below command is an example for Premainnet and an example owner address `e7be097a90c18f6bdd53efe0e74bf34393cac2f0ae941523ea196a47b6859edb`. For other networks, use the appropriate REST URL for the `--url` field. See [Aptos Blockchain Deployments](/nodes/aptos-deployments) for `--url` field values. 
+### 1. Initialize Aptos CLI
+
+  ```bash
+  aptos init --profile mainnet-operator \
+  --private-key <operator_account_private_key> \
+  --rest-url https://fullnode.mainnet.aptoslabs.com/v1 \
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md:35]: This is weird how some say mainnet others say testnet

File: developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md [hunk 3]
```diff
@@ -5,23 +5,118 @@ slug: "staking-pool-operations"
...
+:::
+
+### 2. Check your validator account balance 
+
+Make sure you have some coins to pay gas. You can do this step either by checking on the Aptos Explorer or using the CLI:
+
+On the Aptos Explorer `https://explorer.aptoslabs.com/account/<account-address>?network=testnet` or use the CLI:
+
+```bash
+aptos account list --profile mainnet-operator
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md:50]: --query balance

File: developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md (hunk 2 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md:35]: Fixed.

File: developer-docs-site/docs/nodes/validator-node/operator/shutting-down-nodes.md (hunk 1 above)
Comment (rajkaramchedu) [reply developer-docs-site/docs/nodes/validator-node/operator/shutting-down-nodes.md]: Fixed.

File: developer-docs-site/docs/concepts/staking.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ The following is a high-level description of how validation works on the Aptos b
...
   - The selected leader sends a proposal containing the collected quorum votes of the previous proposal and the leader's proposed order of transactions for the new block. 
-  - All the validators from the validator set will vote on the leader's proposal for the new block. Once a quorum consensus is reached the block can be finalized. Hence the actual list of votes to achieve the quorum consensus is a subset of all the validators in the validator set. This leader validator is rewarded. **Rewards are given only to the leader validators, and not to the voters.**
+  - All the validators from the validator set will vote on the leader's proposal for the new block. Once a quorum consensus is reached the block can be finalized. Hence the actual list of votes to achieve the quorum consensus is a subset of all the validators in the validator set. This leader validator is rewarded. **Rewards are given only to the leader validators, not to the voter validators.**
```
Comment (davidiw) [inline developer-docs-site/docs/concepts/staking.md:86]:
```suggestion
  - All the validators from the validator set will vote on the leader's proposal for the new block. Once consensus is reached the block can be finalized. Hence the actual list of votes to achieve consensus is a subset of all the validators in the validator set. This leader validator is rewarded. **Rewards are given only to the leader validators, not to the voter validators.**
```

Comment (davidiw) [reply developer-docs-site/docs/concepts/staking.md:86]: quorum and consensus seem like interchangeable words

**Review by davidiw — APPROVED** [review]
good for me

**Review by sherry-x — APPROVED** [review]

**Pull request merged**